cartography/intel/azure/iam.py

```diff
@@ -336,8 +336,12 @@ def get_roles_list(client: AuthorizationManagementClient) -> List[Dict]:
             result = client.role_definitions.get_by_id(role["role_definition_id"], raw=True)
             result = result.response.json()
             role['roleName'] = result['properties']['roleName']
```

Comment:
instead of accessing the properties directly, let's use a get based approach to avoid any runtime errors.
so, this line can become: `result.get('properties',{}).get('roleName')`